pyanaconda/modules/payloads/source/source_base.py

```diff
@@ -84,3 +89,43 @@ def tear_down_with_tasks(self):
         :rtype: list[task]
         """
         pass
+
+
+class MountingSourceBase(PayloadSourceBase, ABC):
+
+    counter = 0
+
+    def __init__(self):
+        super().__init__()
+        self._counter_value = MountingSourceBase.counter
+        self._mount_point = "/mnt/anaconda-sources/{}-{:0>4}".format(self.type.value.lower(),
+                                                                     self.counter)
+        MountingSourceBase.counter = MountingSourceBase.counter + 1
```

Comment:
I think that we want to store the id but not here but in the base class. It could be usable for plenty of other things (including logging) in the future and it would be great to have ID for every source. What do you think?

Reply:
Do we want to decide that now? I agree keeping an ID would be somehow "proper", but  at this moment it is not really needed, and the mount point acts as a kind of ID, even if that is rather crude.

Reply:
If we want to have ID for every source, than it is not definitely going to be `counter_value` in `MountingSourceBase`, so please remove that attribute. We can easily modify the code for generating mount points with this future ID.

Reply:
Removed already - somehow GitHub keeps showing the old diff even if the other lines changed :(

In terms of keeping the ID, leaving that open for the future.